source/jormungandr/jormungandr/street_network/handimap.py

```diff
@@ -117,19 +114,15 @@ def status(self):
             },
         }
 
-    def _get_language(self, language):
-        try:
-            return Languages[language].value
-        except KeyError:
-            self.log.error(
-                'Handimap parameter language={} is not a valid parameter - language is set to french by default'.format(
-                    language
-                )
-            )
-            return Languages.french.value
+    def _get_language(self, language_value):
+        language = LANGUAGE_TRANSFORMATION_LIST.get(language_value)
+        if not language:
+            self.log.error('Handimap parameter language={} Invalid - fallback to english'.format(language_value))
+            language = "en-US"
+        return language
 
     def get_language_parameter(self, request):
-        language = request.get('_handimap_language', None)
+        language = request.get('language')
         return self.language if not language else self._get_language(language.lower())
```

Comment:
Since I'm still around :sweat_smile: 
I think you  want to remove the `lower()` call here, there is no way a lowered language tag could ever match something in ` LANGUAGE_TRANSFORMATION_LIST`.

Reply:
Thanks a lot